lib/gollum-lib/filter/code.rb

```diff
@@ -82,17 +82,21 @@ def process(data)
         hl_code = lexer.highlight(code, :options => { :encoding => encoding.to_s, :startinline => true })
       else # Rouge
         begin
+          lexer = Rouge::Lexer.find_fancy(lang || 'plaintext')
+          formatter = Rouge::Formatters::HTML.new
+
           # if `lang` was not defined then assume plaintext
           # if `lang` is defined but cannot be found then wrap it and escape it
-          lang ||= 'plaintext'
-          if Rouge::Lexer.find(lang).nil?
-            lexer     = Rouge::Lexers::PlainText.new
-            formatter = Rouge::Formatters::HTML.new(:wrap => false)
-            hl_code   = formatter.format(lexer.lex(code))
-            hl_code   = "<pre class='highlight'><span class='err'>#{CGI.escapeHTML(hl_code)}</span></pre>"
+          if lexer.nil?
```

Comment:
Would this ever evaluate to true, given `Rouge::Lexer.find_fancy(lang || 'plaintext')` from line 85?